prow/config.yaml

```diff
@@ -103,6 +103,104 @@ presubmits:
       run_if_changed: '^(cmd/kubeadm|build/debs).*$'
       rerun_command: "/test pull-kubernetes-e2e-kubeadm-gce"
       trigger: "(?m)^/test pull-kubernetes-e2e-kubeadm-gce,?(\\s+|$)"
+      skip_branches:
+      - release-1.4
+      - release-1.5
+      - release-1.6  # doesn't have BUILD files under //vendor/k8s.io
```

Comment:
we can probably unskip these now? especially in the chains that only run against some specific branch

Reply:
the unit test complains if I don't have this here. I don't understand whether the unit test is right.

Reply:
hmm that seems wrong (the test). this should fix the need to avoid those branches since BUILD should work fine?
We can probably come back to that in a follow-up.

Reply:
I don't understand your comment here. I think the unit test is right - we have two kubeadm jobs now, one for release-1.6, one for everything else, just like the bazel-build jobs.

Reply:
<img src="https://user-images.githubusercontent.com/917931/31195821-58f99508-a900-11e7-9f0c-681e6a6f9965.png" width="200" title=":man_facepalming:"></img>
G̶i̶t̶H̶u̶b̶ ̶i̶s̶ I am bad at diff. Please ignore.